skopt/optimizer/optimizer.py

```diff
@@ -467,7 +471,7 @@ def tell(self, x, y, fit=True):
                 # minimization starts from `n_restarts_optimizer` different
                 # points and the best minimum is used
                 elif self.acq_optimizer == "lbfgs":
-                    x0 = X[np.argsort(values)[:self.n_restarts_optimizer]]
+                    x0 = X[np.argsort(values)]
```

Comment:
Ok. I will add a comment to jog our collective memory for next time.

I wonder if we always need this many points to kickstart LBFGS. Feels like we should be able to scale this by the number of dimensions instead of using a flat number. (In 3D after testing 10k points do we really still need to LBFGS??)

The other tradeoff we could think about is "we have a budget of (say) 10k evals of the acquisition function, would we find a better point by spending 10k/`N` evals inside `N` LBFGS loops instead of searching at random?"